tidb-in-kubernetes/local-pv-management.md

```diff
@@ -0,0 +1,18 @@
+---
+title: 本地 PV 管理
+category: how-to
+---
+
+# 本地 PV 管理
+
+TiDB 是一款高可用的数据库。TiDB 的存储层 TiKV 对数据进行复制，容忍节点不可用。TiKV 使用高
+ IOPS 和高吞吐量的本地存储，比如 Local SSDs 时，可提高数据库性能。
```

Comment:
Please combine this line the previous one. DO NOT manually break a line. Thanks!